FirebaseStorage/Sources/Storage.swift

```diff
@@ -24,7 +24,7 @@ import FirebaseAuthInterop
 #endif
 
 // Avoids exposing internal FirebaseCore APIs to Swift users.
-@_implementationOnly import FirebaseCoreExtension
+import FirebaseCoreExtension
```

Comment:
Revert when approved:
```suggestion
@_implementationOnly  import FirebaseCoreExtension
```

Reply:
Failure is intentional, from https://github.com/firebase/firebase-ios-sdk/actions/runs/3160910703/jobs/5145901020:
```bash
> Run scripts/check.sh --test-only
START_REVISION=origin/master
START_SHA=018fd[7](https://github.com/firebase/firebase-ios-sdk/actions/runs/3160910703/jobs/5145901020#step:6:8)ec73f[9](https://github.com/firebase/firebase-ios-sdk/actions/runs/3160910703/jobs/5145901020#step:6:10)37c0ddeb2ae1bd27b9cf070b5b4c
Found: clang-format version 14.0.6
+ python --version
Python 3.6.15
+ /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_whitespace.sh
+ /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_filename_spaces.sh
+ /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_copyright.sh
+ /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_test_inclusion.py
+ /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_imports.swift
Import Error: /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/FirebaseStorage/Sources/Storage.swift:26 Use `@_implementationOnly import FirebaseCoreExtension` when importing `FirebaseCoreExtension`.
```